docs/doxygen/overviews/helloworld.h

```diff
@@ -163,6 +149,32 @@ translated) labels and also standard accelerators correct for the current
 platform making your program behaviour more native. For this reason you should
 prefer reusing the standard ids (see @ref page_stockitems) if possible.
 
+We also have to actually connect our event handlers to the events we want to
+handle in them, by calling Bind() to send all the menu events, identified by
+wxEVT_MENU event type, with the specified ID to the given function. The
+parameters we pass to Bind() are
+
+-# The event type, e.g. wxEVT_MENU or wxEVT_BUTTON or wxEVT_SIZE or one
+   of many other events used by wxWidgets.
+-# Pointer to the member function to call and the object to call it on. In
+   this case we just call our own function, but we could call a member
```

Comment:
... call our own function (hence 'this' for the object), but we could call ... maybe it is too obvious, but personally I think it's worth being clear why there is a 'this' in the call.